twoliter/embedded/imghelper

```diff
@@ -1,6 +1,150 @@
 #!/usr/bin/env bash
-
 # shellcheck disable=SC2034 # variables are used through parameter substitution
+
+DATA_IMAGE_SUFFIX="-data"
+BOOT_IMAGE_SUFFIX="-boot.ext4.lz4"
+VERITY_IMAGE_SUFFIX="-root.verity.lz4"
+ROOT_IMAGE_SUFFIX="-root.ext4.lz4"
```

Comment:
Another weird behavior inherited from `rpm2img`. It was probably written this way since the extension for data depends on the build args whereas the others are constant. We can drop them and add the extensions to the `symlink_image` calls, it just means a bigger delta from the original script.